applications/migrations/0018_auto_20160628_2154.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.7 on 2016-06-28 21:54
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def deduplicate_applications(apps, schema_editor):
+    Application = apps.get_model("applications", "Application")
+    applications = Application.objects.values_list("form", "email").distinct()
+
+    for form, email in applications:
+        duplicates = (
+            Application.objects
+            .filter(form=form, email=email)
+            .values_list('id', flat=True)
+            .order_by('created')[1:]
+        )
+        Application.objects.filter(id__in=list(duplicates)).delete()
```

Comment:
Oh this could be fixed by filtering by `('state', 'created')`, as accepted application will have state as `accepted`, which is first alphabetically from all available states :)